test/react-native/features/react-native-navigation.feature

```diff
@@ -1,32 +1,51 @@
 @react_native_navigation
-Feature: Navigation spans with React Native Navigation
+Feature: React native navigation support
 
-  Scenario: Navigation Spans are automatically instrumented
+  Scenario: Navigation spans are automatically instrumented
     When I run 'ReactNativeNavigationScenario'
     And I wait to receive a sampling request
     And I wait for 1 span
 
-    # Check the initial probability request
-    Then the sampling request "Bugsnag-Span-Sampling" header equals "1.0:0"
-
-    And the trace "Bugsnag-Span-Sampling" header equals "1:3"
-
-    And a span named "[AppStart/ReactNativeInit]" contains the attributes:
```

Comment:
what's the reason for removing the app start test?

Reply:
ahh, I was just going to separate it out into a different scenario, but it appears I've missed the 'adding it back in' part

Reply:
I've now updated the feature to run the app start span scenario